### PR TITLE
Fix: replaced buttons with custom buttons

### PR DESCRIPTION
**Pull request by guacamole**
Signed-off-by: guacamole <gunjanwalecha@gmail.com>

### CODE REVIEW

File: src/components/newRepository.svelte [hunk 1]
```diff
@@ -27,21 +26,15 @@
...
-			<div class="flex px-16 pt-8 w-full">
-				<Button styles="text-gray-50 w-full mr-2" label="Create" />
-				<Button
-					onClick={() => toggleModal()}
-					styles="bg-gray-50 text-gray-800 w-2/3 ml-2"
-					label="Close"
-				/>
+			<div class="flex px-16 pt-8 w-full justify-center items-center space-x-5">
+				<ButtonSolid>Create</ButtonSolid>
+				<ButtonOutlined onClick={() => toggleModal()}>Close</ButtonOutlined>
```
Comment (jay-dee7) [inline src/components/newRepository.svelte:37]: @guacamole we don't need the arrow func here, just `toggleModal` would do

**Review by jay-dee7 — CHANGES_REQUESTED** [review]
@guacamole just some tiny change is needed for this to be merged

File: src/components/newRepository.svelte (hunk 1 above)
Comment (guacamole) [reply src/components/newRepository.svelte]: you got it!

**Review by jay-dee7 — APPROVED** [review]
LGTM 🥇 🥇 🥇

**Pull request merged**